lib/bunyan_helper.js

```diff
@@ -43,8 +51,9 @@ function appendStream(streams, s) {
  * A Bunyan stream to capture records in a ring buffer and only pass through
  * on a higher-level record. E.g. buffer up all records but only dump when
  * getting a WARN or above.
- *
- * @param {Object} options contains the parameters:
+ * @public
+ * @class
+ * @param {Object} opts contains the parameters:
  *      - {Object} stream The stream to which to write when dumping captured
```

Comment:
Good catch, that was an existing comment block, so I didn't look too closely. I'll fix it up as proposed.